package.json

```diff
@@ -53,10 +53,11 @@
 		"php:autofix": "composer php:autofix",
 		"test-adminpage": "yarn test-client && yarn test-gui",
 		"test-client": "NODE_ENV=test NODE_PATH=tests:_inc/client:node_modules/@automattic/dops-components/client tests/runner.js",
+		"test-extensions": "jest",
 		"add-textdomain": "wpi18n addtextdomain --textdomain=jetpack --glob-pattern='!(docker|node_modules|tests|tools|vendor){*.php,**/*.php}'",
 		"test-gui": "NODE_ENV=test NODE_PATH=tests:_inc/client:node_modules/@automattic/dops-components/client tests/runner.js gui",
 		"test-modules": "NODE_ENV=test NODE_PATH=tests:_inc/client:node_modules/@automattic/dops-components/client tests/runner.js modules",
-		"test-dangerci-and-adminpage": "yarn danger ci && yarn test-adminpage",
+		"test-dangerci-and-adminpage-and-extensions": "yarn danger ci && yarn test-adminpage && yarn test-extensions",
```

Comment:
Works as is, kinda feels like adding [`npm-run-all`](https://www.npmjs.com/package/npm-run-all) to run these in parallel would be nicer for CI speed:

```
"test-dangerci-and-adminpage-and-extensions": "run-p 'danger ci' test-adminpage test-extensions"
```

Reply:
I almost commented about how ugly this is 😅 It is just extending the existing system, so I'm inclined to leave it this way for this PR.

> to run these in parallel would be nicer for CI speed

Parallelizing work on a single machine in CI doesn't typically show much improvement in my experience. I'm not sure with our current Travis setup in Jetpack, but I don't expect they're well equipped to handle a lot of parallelization, I wouldn't be surprised if it resulted in worse overall performance.